### PR TITLE
Adding first draft for audit logging

### PR DESCRIPTION
**Pull request by poojaSprophecy**

### CODE REVIEW

**Comment by prashanth26** [comment]
Hi @poojaSprophecy ,

I have added a section on [customer steps](https://github.com/SimpleDataLabsInc/prophecy-docs/pull/134/commits/78765330ddd05ed88ec35707244c8f89da26f0d3) in the same doc. Please feel free to update it as you may think fit. 

You will find more details on the approach used in our internal doc which I have DMed you.

File: docs/metadata/configure-audit-logging.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+| Project    | addProject                          | Add a project                                       | ["name","forkMode","language", "ownerUid", "mainBranchModificationAllowed"]                                          |
+|            | getDetails                          | Get Details of a Project                            | ["projectId"]                                                                                                        |
+|            | Project List Query                  | List all projects for User                          | ["uid"]                                                                                                              |
+|            | teamProjectAvailable                | Available Projects for that Team                    | ["uid", "language"]                                                                                                  |
+|            | add Project Dependency              | Add a dependency Project to Current                 | ["projectId", "DependencyProjectUid"]                                                                                |
```
Comment (several27) [inline docs/metadata/configure-audit-logging.md:48]: How come some of the queries have spaces and some not? Some are uppercase and some not? Can we clean this up?

File: docs/metadata/configure-audit-logging.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+title: Configure Audit Logging
+id: auditLogging
+description: Prophecy Audit logs available to be exported in S3
+sidebar_position: 6
+tags:
+  - Audit Logs
+---
+
+Prophecy provides access to audit logs of activities performed by Prophecy users, allowing your enterprise to monitor detailed usage patterns.
+User can configure a S3 bucket to sync these events from Prophecy to their environment.
```
Comment (several27) [inline docs/metadata/configure-audit-logging.md:11]: The Prophecy admin~~User~~ can configure...

File: docs/metadata/configure-audit-logging.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+## Configure S3 bucket for logs
+
+1. Create/reuse an AWS IAM user who requires access to Prophecy audit logs. There is no need to attach any policies to this user.
+2. Share the ARN of the AWS user with Prophecy, inorder to grant permissions to access the audit events stored in Prophecy S3 bucket. The ARN should be of the format `arn:aws:iam::{{Customer-Account-ID}:user/{Customer-User}` with the actual customer details. Eg: `arn:aws:iam::123456789012:user/bob`.
```
Comment (several27) [inline docs/metadata/configure-audit-logging.md:16]: in order (missing space)

File: docs/metadata/configure-audit-logging.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+description: Prophecy Audit logs available to be exported in S3
+sidebar_position: 6
+tags:
+  - Audit Logs
+---
+
+Prophecy provides access to audit logs of activities performed by Prophecy users, allowing your enterprise to monitor detailed usage patterns.
+User can configure a S3 bucket to sync these events from Prophecy to their environment.
+
+## Configure S3 bucket for logs
```
Comment (several27) [inline docs/metadata/configure-audit-logging.md:13]: We're missing a little bit of an introduction here. It should answer questions like: Is this for Public SaaS only or all environments? High-level process description. Is it manual?

File: docs/metadata/configure-audit-logging.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+1. Create/reuse an AWS IAM user who requires access to Prophecy audit logs. There is no need to attach any policies to this user.
+2. Share the ARN of the AWS user with Prophecy, inorder to grant permissions to access the audit events stored in Prophecy S3 bucket. The ARN should be of the format `arn:aws:iam::{{Customer-Account-ID}:user/{Customer-User}` with the actual customer details. Eg: `arn:aws:iam::123456789012:user/bob`.
+3. Reach out to Prophecy at [contact us](mailto:success@Prophecy.io) with above details to enable this in your account.
+4. Prophecy would provide the required access and communicate back the S3 bucket URL that can be used to access the events.
+5. You may now be able to access this bucket over an application (or) AWS CLI using the above user credentials.
+6. Example of AWS CLI access is given below. Use the below command to get the latest events bucket locally. You may use similar mechanism to access it from your applications.
```
Comment (several27) [inline docs/metadata/configure-audit-logging.md:20]: Could @manjunathprophecy put together a summary of setting up Splunk integration on s3? Or maybe link to some tutorial. I'm sure Workday will need help.

**Review by several27 — DISMISSED** [review]

File: docs/metadata/configure-audit-logging.md (hunk 4 above)
Comment (manjunathprophecy) [reply docs/metadata/configure-audit-logging.md:13]: This applies for Public SaaS only .. Right now its manual from both customer/Our side .. We will add these details

File: docs/metadata/configure-audit-logging.md (hunk 5 above)
Comment (manjunathprophecy) [reply docs/metadata/configure-audit-logging.md:20]: We have not tried Yet this Audit log by integrating into Splunk ,  We will setup one for within our env. and later we can document ?  and add it here

File: docs/metadata/configure-audit-logging.md (hunk 1 above)
Comment (poojaSprophecy) [reply docs/metadata/configure-audit-logging.md]: fixed.

**Review by jainabhinav — APPROVED** [review]

**Pull request merged**